sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.controller.service.ts

```diff
@@ -0,0 +1,100 @@
+import { Injectable, NotFoundException } from "@nestjs/common";
+import { ApplicationOfferingChangeRequestService } from "../../services";
+import {
+  ApplicationOfferingChangesAPIOutDTO,
+  ApplicationOfferingDetailsAPIOutDTO,
+} from "./models/application-offering-change-request.dto";
+import { getUserFullName } from "../../utilities";
+
+@Injectable()
+export class ApplicationOfferingChangeRequestControllerService {
+  constructor(
+    private readonly applicationOfferingChangeRequestService: ApplicationOfferingChangeRequestService,
+  ) {}
+
+  /**
+   * Get the Application Offering Change Request by its id for the institution.
+   * @param id the Application Offering Change Request id.
+   * @param options method options:
+   * - `locationId`: location id for institution authorization.
+   * @returns application offering change request.
+   */
+  async getById(
```

Comment:
I get your point, there is nothing wrong in creating a overridden method, but the reason for naming the methods for the process they do is what we are following as a standard. So it would be better to make them explanatory and explaing the paramaters wherever possible. Also the variable names are mentioned as proper applicationOfferingChangeRequestId rather than id, is again for the same purpose and even if the developer misses the comments on the top of the method the calling class will have an idea when we try to implement them.
Also this will create consistency of the existing code.